content/installation/java/configuration/JavaAgentSystemProperties.md

```diff
@@ -75,7 +75,7 @@ By default, diagnostic logging is enabled, but set to the INFO level. It uses a
 |Property|Description|
 |-|-|
 | contrast.level | Log output level<br>**DEFAULT VALUE:** info</br> |
-| contrast.log | Enable *diagnostic logging*. This hurts performance, but generates useful information for debugging Contrast. The value set here will be the location to which log output is saved. If no log file exists at this location, one will be created. For instance, */opt/Contrast/contrast.log* will create a log in the */opt/Contrast* directory and rotate it automatically as needed.<br>**DEFAULT VALUE:** ${HOME}/.contrast/logs/contrast.log</br> |
+| contrast.log | Enable *diagnostic logging*. This hurts performance, but generates useful information for debugging Contrast. The value set here will be the location to which log output is saved. If no log file exists at this location, one will be created. For instance, */opt/Contrast/contrast.log* will create a log in the */opt/Contrast* directory and rotate it automatically as needed.<br>**DEFAULT VALUE:** ${HOME}/.contrast/contrast.log</br> |
```

Comment:
This file is auto-generated based on a backend java file. Creating a ticket to update that file instead since any changes here will be overwritten.